benefits/core/admin/common.py

```diff
@@ -1,8 +1,20 @@
 from django.contrib import admin
+from django.http import HttpRequest
 
 from benefits.core import models
 
 
 @admin.register(models.PemData)
 class PemDataAdmin(admin.ModelAdmin):
-    list_display = ("label",)
```

Comment:
_(This is nitpicky, but important for consistency)_

Django admin by default uses `label` in the table, so the table will display the same either way.

What superusers see:

| With this line | Without this line |
| --- | --- | 
|![image](https://github.com/user-attachments/assets/f7ab89c3-1de4-4d16-8629-e1f98ab1cc27)  |![image](https://github.com/user-attachments/assets/ad3bfa7b-df15-4ffd-9328-229f5a4625e0) |

But since we're adopting a philosophy of being explicit even if it's redundant, I think it'd be better to keep this line.

Reply:
Hmm, just checked through our various `ModelAdmin`s... not all of them define a `list_display`. It makes sense to be explicit even if redundant for the permissions since that logic is complex and varies across user types, but maybe for display it's not as important.

Reply:
That sounds good @angela-tran! If we decide otherwise, I'm happy to bring it back and make it explicit.